src/Compilers/VisualBasic/Test/Emit/Attributes/InternalsVisibleToAndStrongNameTests.vb

```diff
@@ -1660,21 +1662,21 @@ End Class
 
     <Fact>
     Public Sub PublicSign_FromPublicKeyFileNoStrongNameProvider()
-        Dim snk = Temp.CreateFile().WriteAllBytes(TestResources.General.snKey)
```

Comment:
Is ```snKey``` resource used anywhere? We should consider removing it if it is no longer used.